pkgs/development/python-modules/aiosmtpd/default.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, atpublic }:
+
+# python3 only
+buildPythonPackage rec {
+  pname = "aiosmtpd";
+  version = "1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1xdfk741pjmz1cm8dsi4n5vq4517i175rm94696m3f7kcgk7xsmp";
+  };
+
+  propagatedBuildInputs = [
+    atpublic
+  ];
+
+  # error: [Errno 111] Connection refused
```

```suggestion
  # Tests need network access
```
or something along the lines